feature-libs/checkout/components/components/delivery-mode/delivery-mode.component.ts

```diff
@@ -42,7 +45,9 @@ export class DeliveryModeComponent implements OnInit, OnDestroy {
     private checkoutDeliveryService: CheckoutDeliveryFacade,
     private checkoutConfigService: CheckoutConfigService,
     private activatedRoute: ActivatedRoute,
-    protected checkoutStepService: CheckoutStepService
+    protected checkoutStepService: CheckoutStepService,
+    // TODO:#13888 remove after the deprecation is done
+    protected featureConfigService?: FeatureConfigService
```

Comment:
Do we need to do the constructor deprecation thingy? 🤔
`FeatureConfigService` is an optional parameter which will be removed in the future, forcing us to do the constructor deprecation again.